lua/entities/gmod_wire_expression2/core/string.lua

```diff
@@ -36,6 +36,76 @@ end)
 
 /******************************************************************************/
 
+registerOperator("fea", "nss", "", function(self, args)
+	local keyname, valname = args[2], args[3]
+	local sub = utf8.sub
```

Comment:
Should cache this outside of the actual operator

Reply:
n/a